pkgs/development/libraries/openssl/default.nix

```diff
@@ -256,7 +257,29 @@ in {
 
```

Comment:
the sha265 of openssl_1_1 needs to be changed to hash, too

Reply:
Thanks for the hint. It got lost during a rebase.